### PR TITLE
Rewrite without overriding templates

### PR DESCRIPTION
**Pull request by communiteq**
This theme component was broken (bulk select didn't work anymore)
When trying to fix it we saw that there are enough raw plugin outlets to be able to add the column without overriding hbs templates, so the theme component was completely reworked to use raw plugin outlets.

### CODE REVIEW

File: desktop/desktop.scss [hunk 1]
```diff
@@ -1,11 +1,5 @@
-.link-bottom-line {
-  font-size: unset;
-}
-
-.topic-list-item .discourse-tags {
-  display: block;
-  font-size: var(--font-down-2);
-  clear: both;
+.link-bottom-line .badge-wrapper {
```
Comment (nolosb) [inline desktop/desktop.scss:1]: .badge-wrapper was replaced with badge-category__wrapper in https://github.com/discourse/discourse/commit/797da5870be1e651f68d32ec47374a057c878625 (moving to single category badge style)

Comment (communiteq) [reply desktop/desktop.scss]: Thanks - fixed

**Comment by tshenry** [comment]
Thanks for working on this @communiteq!

Would you be able to remove the one locale item in the `/locales/en.yml` file (or the whole file even) since that's now unused? It was originally for explicitly including a column name here:

Screenshot 2024-01-12 at 1 31 20 PM

That's not going to be possible with the new approach so we might as well remove it.

We also lose the ability to sort by the category column which is a little more of a bummer, but again not critical.

I feel like ditching those in exchange for avoiding template overrides is worth it!

**Comment by communiteq** [comment]
I've removed the unused locale file. Seems to me that that should be a separate theme component anyway.

I didn't even realize the ability to sort was gone. 
I have resolved that now but I'm not sure if I did it in the most easy manner so I'm open to your feedback on that 😎

**Review by tshenry — APPROVED** [review]

**Pull request merged**